luatest/replica_set.lua

```diff
@@ -183,12 +183,16 @@ function ReplicaSet:stop()
     end
 end
 
---- Stop all servers in the replica set and clean their working directories.
+--- Stop all servers in the replica set and save their artifacts if the test fails.
+-- This function should be used only at the end of the test (after_test,
+-- after_each, after_all hooks) to terminate all server processes in the
+-- replica set. Besides process termination, it saves the contents of each
+-- server working directory to the <vardir>/artifacts directory for further
```

Comment:
`<vardir>/artifacts` should be wrapped into backticks.

Reply:
Fixed